home.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Anirveda</title>
+    <h1> Anirveda</h1>
+    <h2>The TechnoEconomic Club of PDEU</h2>
+</head>
+<body>
+    <img src="anirveda_logo.png.jpg" id="image"

```

Comment:
You have added 2 file extensions here and also change alt tag text to "Anirveda Logo"